docs/adr/0016-auth-for-settings-redesign.md

```diff
@@ -0,0 +1,114 @@
+# Authentication in Settings Redesign
+
+- Status: proposed
+- Deciders: Les Orchard, Lauren Zugai, Ben Bangert, Ryan Kelly
+- Date: 2020-04-15
+
+Issue: https://jira.mozilla.com/browse/FXA-1559
+
+## Context and Problem Statement
+
+The Settings Redesign app needs to query & mutate the same protected user data as the existing settings app hosted on content-server. This will require some form of authentication & authorization to manage that data. [ADR-0014] included several assumptions that turned out to be inaccurate during implementation efforts. Using OAuth2 tokens in the auth-server for the API calls needed is a larger effort than anticipated and will not meet time/work constraints for the Settings Redesign. This ADR includes a new option that is proposed to meet these decision drivers.
+
+## Decision Drivers
+
+- Smooth UX
+- Security
+- Development velocity
+- Ease of integration
+
+## Considered Options
```

Comment:
Looks like these are a copy of the superseded ADR. Might help to just discuss the options with updated discoveries. There aren't new details surfaced about the previously rejected ones, so they don't seem to factor into the decision here.

Reply:
Hmm yeah I agree, this ADR could just be the old approach from ADR 14 vs the new proposed one with pros/cons of each direction, and the decision drivers can be the reasons for the new approach.